.vscode/settings.json

```diff
@@ -29,6 +29,7 @@
         ".vscode",
         ".vscode-insiders"
     ],
+    "markdown.extension.toc.levels": "2..3",
```

Comment:
Specify the heading level at which the table of contents will be automatically created.